modules/core/src/test/scala/org/scalasteward/core/coursier/CoursierAlgTest.scala

```diff
@@ -0,0 +1,22 @@
+package org.scalasteward.core.coursier
+
+import org.scalasteward.core.data.Dependency
+import org.scalasteward.core.mock.MockContext._
+import org.scalasteward.core.mock.MockState
+import org.scalatest.{FunSuite, Matchers}
+
+class CoursierAlgTest extends FunSuite with Matchers {
+
+  test("getArtifactUrl") {
+    val dep = Dependency("org.typelevel", "cats-effect", "cats-effect_2.12", "1.0.0")
+    coursierAlg
+      .getArtifactUrl(dep)
+      .runS(MockState.empty)
```

Comment:
I'm a bit confused by this test. It doesn't seem to be testing anything. Don't you want to call `runA(MockState.empty)` and then assert that the artifact URL is what you expect?

Reply:
Thanks !
I had no idea how to get result, not `state`, but now figured out.
Added assertion in `https://github.com/fthomas/scala-steward/pull/762/commits/98521746ee7aefbd8b55ddc3c88214a142616236